blockchain/beacon/src/drand.rs

```diff
@@ -34,6 +35,40 @@ impl DrandPublic {
     }
 }
 
+pub struct Schedule<T>(pub Vec<BeaconPoint<T>>)
+where
+    T: Beacon;
```

Comment:
you shouldn't need to constrain T here

Reply:
Just following lotus implementing in which the beacon can be switched around. Since it is all at compile time, don't see the issue but will change

Reply:
Yeah, you just don't need to constrain it _there_. It should be constrained only on the implementation that requires it